src/Sql/Sql/Replication/Services/AzureSqlDatabaseReplicationAdapter.cs

```diff
@@ -392,7 +392,7 @@ private AzureReplicationLinkModel CreateReplicationLinkModelFromResponse(string
             model.ServerName = serverName;
             model.DatabaseName = databaseName;
             model.AllowConnections = allowConnections;
-            model.Location = resp.Location;
+            model.Location = GetServerLocation(resourceGroupName, serverName);
```

Comment:
i believe this should be resp.location still

Reply:
this is removed in the > 1.50 version of SDK